### PR TITLE
✨ Brutalist Revamp 🧱

### PR DESCRIPTION
**Pull request by Sak1012**
**Primitive af 💥**

### CODE REVIEW

**Comment by Sak1012** [comment]
image
image
image
image
image

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+yarn.lock
```
Comment (MKMukeshkannan) [inline .gitignore:38]: ⚠️ ⚠️  Kindly don't do this

File: src/app/team/page.tsx [hunk 2]
```diff
@@ -148,7 +148,7 @@ const HoverComponent = ({ name, post }: Img) => {
 export default function Page() {
 	return (
 		<>
-			<main className="flex flex-col justify-center items-center px-8 graph-paper">
+			<main className="flex flex-col justify-center bg-[#fbfbf2] items-center px-8 ">
```
Comment (MKMukeshkannan) [inline src/app/team/page.tsx:151]: This color looks good, 
but overall the Hero page looks deserted. 
try increasing font size.

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -55,27 +55,27 @@ const legalDuck: Array<DuckElement> = [
 
 export default function Footer() {
 	return (
-		<footer className="overflow-clip bg-[#b3a3f3] relative">
```
Comment (MKMukeshkannan) [inline src/components/Footer.tsx:58]: kindly stick with the accent color

File: src/components/Main/index.tsx [hunk 4]
```diff
@@ -29,107 +25,52 @@ export default function Main() {
...
-
-				<section className=" flex-[1] relative border-x border-black min-h-screen hidden lg:block">
-					<Image
-						src={AboutUSCover}
-						alt="about-us-cover"
-						fill
-						className="object-cover"
-					/>
-				</section>
+				<p className="text-2xl md:text-5xl leading-loose font-inter text-justify">
```
Comment (MKMukeshkannan) [inline src/components/Main/index.tsx:57]: I personally didn't like this one,
but rahul m naveeth is fine with it.

**Review by MKMukeshkannan — CHANGES_REQUESTED** [review]

**Comment by MKMukeshkannan** [comment]
Kindly do something to hero section.

**Pull request closed without merging**